src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -347,6 +361,20 @@ public function getFormatToMimeTypeMapProvider()
         );
     }
 
+    public function getFormatToMimeTypeMapProviderForAccept()
+    {
+        return array(
+            array(null, array(null, 'unexistent-mime-type'), null),
+            array('txt', array('text/plain'), 'text/plain'),
+            array('js', array('application/javascript', 'application/x-javascript', 'text/javascript'), 'text/javascript'),
+            array('css', array('text/css'), 'text/css'),
+            array('json', array('application/json', 'application/x-json', 'application/json;version=1.0', 'application/json;version=1.1'), 'application/json;version=1.0'),
+            array('xml', array('text/xml', 'application/xml', 'application/x-xml', 'application/xml;version=1.0', 'application/xml;version=2.0'), 'application/xml;version=2.0'),
```

Comment:
The two test cases above look wrong to me. The `version` attribute should not be returned by the `getMimeType()` method (it should still be `application/json`).